### PR TITLE
[vlive] Update extraction

### PR DESCRIPTION
**Pull request by coreynicholson**
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix

The format of the website has changed slightly breaking the extractor, this fixes it.

### CODE REVIEW

File: youtube_dl/extractor/vlive.py [hunk 1]
```diff
@@ -51,14 +51,12 @@ def _real_extract(self, url):
         video_params = self._search_regex(
             r'\bvlive\.video\.init\(([^)]+)\)',
             webpage, 'video params')
-        status, _, _, live_params, long_video_id, key = re.split(
-            r'"\s*,\s*"', video_params)[2:8]
+        status, _, _, long_video_id, key = [p.strip(r'"') for p in re.split(
+            r'\s*,\s*', video_params)[2:7]]
```
Comment (fstirlitz) [inline youtube_dl/extractor/vlive.py]: Actually, why not just do
```py
        video_params = self._parse_json(self._search_regex(
            r'\bvlive\.video\.init\(([^)]+)\);',
            webpage, 'video params'), video_id,
            transform_source=lambda s: '[' + s + ']')

        status, long_video_id, key = video_params[2], video_params[6], video_params[7]
```
It should be much more robust.

Comment (coreynicholson) [inline youtube_dl/extractor/vlive.py]: Sure, I've updated it.

**Comment by Kagami** [comment]
Anything prevents this to be merged? Ping @yan12125 @dstftw 

**master**

```
$ python -m youtube_dl -F http://www.vlive.tv/video/16189
[vlive] 16189: Downloading webpage
[vlive] 16189: Downloading JSON metadata
ERROR: No video formats found; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

**remotes/coreynicholson/master**

```
$ python -m youtube_dl -F http://www.vlive.tv/video/16189
[vlive] 16189: Downloading webpage
[vlive] 16189: Downloading JSON metadata
[info] Available formats for 16189:
format code  extension  resolution note
270P         mp4        480x270     464k video@ 368k, audio@ 96k, 15.59MiB
360P         mp4        640x360     726k video@ 598k, audio@128k, 24.32MiB
480P         mp4        854x480     914k video@ 786k, audio@128k, 30.57MiB
720P         mp4        1280x720   1651k video@1459k, audio@192k, 55.14MiB
1080P        mp4        1920x1080  3317k video@3125k, audio@192k, 110.64MiB (best)
```

Simple test works for me.

**Comment by coreynicholson** [comment]
Thanks @Kagami, it's my first pull request so I didn't want to be too pushy, but really no idea what's preventing the merge, a new melon extractor got merged yesterday same day as the request was opened yet this fix for broken functionality is being delayed.

**Comment by dstftw** [comment]
No live streams were available at the moment of testing thus not merged.

**Comment by Kagami** [comment]
Yeah, testing live streams is a pain. There are few lives at the moment:

```
$ python -m youtube_dl http://www.vlive.tv/video/18379 -F                                       
[vlive] 18379: Downloading webpage
[vlive] 18379: Downloading webpage
[vlive] 18379: Downloading m3u8 information
[vlive] 18379: Downloading m3u8 information
[vlive] 18379: Downloading m3u8 information
[vlive] 18379: Downloading m3u8 information
[vlive] 18379: Downloading m3u8 information
[info] Available formats for 18379:
format code  extension  resolution note
180P-meta    mp4        multiple   Quality selection URL 
360P_-meta   mp4        multiple   Quality selection URL 
360P-meta    mp4        multiple   Quality selection URL 
720P-meta    mp4        multiple   Quality selection URL 
audio-meta   mp4        multiple   Quality selection URL 
audio        mp4        audio only  112k , mp4a.40.2
180P         mp4        320x184     480k , avc1.77.21, mp4a.40.2
360P         mp4        640x368     849k , avc1.77.30, mp4a.40.2
360P_        mp4        640x368    1685k , avc1.77.32, mp4a.40.2
720P         mp4        1280x720   2534k , avc1.77.41, mp4a.40.2 (best)
```

**Comment by yan12125** [comment]
For future references, vlive schedule: https://github.com/rg3/youtube-dl/issues/9321#issuecomment-215200950

**Comment by coreynicholson** [comment]
A lot of vlives are spontaneous and unscheduled so the easiest way is to just check the homepage (http://www.vlive.tv/), it will display all the live streams that are currently happening. They happen quiet often during the afternoon and evening hours in korea.

**Comment by dstftw** [comment]
Do you want to be credited as one of the authors of youtube-dl? If so under what name?

**Pull request closed without merging**